docker/runtimes/action-python-3/Dockerfile

```diff
@@ -1,8 +1,45 @@
-FROM openwhisk/python3action:1.0.0
+FROM jboss/base:latest
 
-# OpenShift compatibility
-RUN chgrp -R 0 /pythonAction \
+USER root
+
+RUN yum -y install centos-release-scl-rh gcc libxslt-devel libxml2-devel \
+  && yum -y install rh-python36 rh-python36-python-pip rh-python36-python-setuptools rh-python36-python-six rh-python36-python-devel \
+  && scl enable rh-python36 "pip3 install --no-cache-dir \
+        gevent==1.1.2 \
+        flask==0.11.1 \
+        beautifulsoup4==4.5.1 \
+        httplib2==0.9.2 \
+        kafka_python==1.3.1 \
+        lxml==3.6.4 \
+        python-dateutil==2.5.3 \
+        requests==2.11.1 \
+        scrapy==1.1.2 \
+        simplejson==3.8.2 \
+        virtualenv==15.1.0 \
+        twisted==16.4.0" \
+  && yum -y remove gcc libxslt-devel libxml2-devel \
+  && yum clean all \
+  && rm -rf /var/cache/yum
+
+ENV FLASK_PROXY_PORT 8080
+
+ADD https://raw.githubusercontent.com/apache/incubator-openwhisk-runtime-docker/dockerskeleton%401.1.0/core/actionProxy/actionproxy.py /actionProxy/actionproxy.py
```

Comment:
In other images, we clone/checkout, e.g. https://github.com/projectodd/openwhisk-openshift/blob/master/docker/alarms/Dockerfile#L14-L18

But I'm not sure one way is any better than another. :)

Reply:
Using `ADD` seems simpler, we don't need to install git inside. We can even use GH feature of generating the ZIP for us and it will be unpacked on the fly by the `ADD` command.

Do we need `git` inside of the image?

Reply:
Nah, I agree. No reason to install git.